.github/workflows/build.yaml

```diff
@@ -30,3 +30,4 @@ jobs:
           name: coverage
           path: htmlcov
           if-no-files-found: error
+
```

Comment:
It seems that some files are changed with additional new lines, and others have no new lines.  Is this a feature of some IDE?